src/components/tooltip/__stories__/index.story.jsx

```diff
@@ -0,0 +1,52 @@
+import * as React from 'react';
+import { Tooltip, Button } from '@/index';
+
+export const Basic = () => {
+  const options = {
+    tooltip: 'An awesome tooltip',
+    hoverable: true,
```

Comment:
Please mention `hoverable` prop in the prop table. Also mention all the options user can pass to tooltip inside prop table